README.md

```diff
@@ -14,3 +14,48 @@ These attributes differentiate V1 from V2:
 ## When will it be available? 
 
 * We are targetting Firefox 62 (nightly) to have the new APIs available
+
+## What will the API Look like? 
+
+There are two new APIs to work with settings:
+
+1. `get()` 
+2. `on()` 
+
+### get()
+
+```js
+const { RemoteSettings } = ChromeUtils.import("resource://services-common/remote-settings.js", {});
+
+const records = await RemoteSettings("my-key").get();
```

Comment:
`records` was intentional here? Maybe `entries` to be consistent with the `for` loop?

Reply:
it was intentional. I thought it would be clearer. I don't mind changing it to `entries`